index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      property="og:image"
+      content="https://user-images.githubusercontent.com/79058364/154861184-fee4bb8d-657f-47e2-aff2-ea3c72a2a64f.png"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Redefining The Future</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./CSS/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="social_media">
+        <ul class="media-nav d-flex flex-center">
+          <li class="media_link">
+            <a href="#"><i class="fa fa-facebook"></i></a>
+          </li>
+          <li class="media_link">
+            <a href="https://twitter.com/Evans_22J"
+              ><i class="fa fa-twitter"></i
+            ></a>
+          </li>
+          <li class="media_link"><a href="#">English</a></li>
+          <li class="media_link"><a href="#">My page</a></li>
+          <li class="media_link"><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <nav class="menu">
+        <div class="hamburger d-flex flex-center">
+          <img
+            src="./images/icons/Union.png"
+            class="show-menu-btn"
+            alt="union"
+          />
+          <img
+            src="./images/icons/Icon@2x.svg"
+            class="closed-menu-btn"
+            alt="icons"
+          />
+          <div class="logo">
+            <a href="./about.html"
+              ><img src="./images/icons/test-africa.png" alt="logo"
+            /></a>
+          </div>
+          <ul class="nav_menu d-flex">
+            <li class="menu_link active"><a href="./about.html">About</a></li>
+            <li class="menu_link"><a href="#Program">News</a></li>
+            <li class="menu_link"><a href="#Team">Team</a></li>
+            <li class="menu_link"><a href="#Partner">Partners</a></li>
+            <li class="menu_link active"><a href="#Africa">Africa</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="Home" class="home_section">
+        <div class="home_section_title">
+          <h2>"Redefining The Future"</h2>
+          <h1>AFRICA MUST THINK ENTREPRENEURSHIP CONFERENCE - 2022</h1>
```

Comment:
[OPTIONAL] It would be nicer if you remove the scrolling bar from the title to improve the UI design.

<img width="944" alt="image" src="https://user-images.githubusercontent.com/77190203/159017828-a0bb249e-2e3e-4b17-b21d-7e38e774dbe2.png">

Reply:
I have tried browsing on 3 browsers with 3 different laptops I'm unable to find that line including live view in vs code